x-team/auto-ui
# Include ranking column on ApplicantBadge

**Release Type:** *New feature*

Fixes https://x-team-internal.atlassian.net/browse/XP-2261

## Description

This PR includes the ranking value in the ApplicantBadge as an input field of type number with min value 1 and max value 10.

## Checklist

- [x] if any snapshots have been changed, verify that component still works and looks as expected and update changed snapshot

## Related PRs

branch | PR
------ | ------
`XP-2261-include-ranking-column` | [Auto](https://github.com/x-team/auto/pull/1593)
`XP-2244-ranking-column-API` | [Auto API](https://github.com/x-team/auto-api/pull/529)

## Steps to Test or Reproduce

1. Check ApplicantBadge component on tabular view mode
1. Make sure the prop `ranking` is set with a numeric value
1. Observe the input field appears with the defined value

## Impacted Areas in Application

- ApplicantBadge component

## Screenshots

<img width="1593" alt="screen shot 2018-10-16 at 7 29 52 pm" src="https://user-images.githubusercontent.com/131859/47051255-e4857680-d179-11e8-8b1d-a14ea8e8958f.png">
